test-clients/src/main/java/com/hedera/services/yahcli/README.md

```diff
@@ -0,0 +1,222 @@
+                          'contract', 'scheduled')
+                        or 'all' to get fees for all basic operations
+  Commands:
+    help  Displays help information about the specified command
+```
+
+Example Run:
+```
+$ docker run -v $(pwd):/launch gcr.io/hedera-registry/yahcli:0.0.1 -n localhost -p 56 fees list-base-prices crypto
+
+-------------------------------|-----------------|
+    Transaction and Query Fees |                 |
+-------------------------------|-----------------|
+-------------------------------|-----------------|
+        Cryptocurrency Service | Fees            |
+-------------------------------|-----------------|
+                  cryptoCreate | 0.04992         |
+                  cryptoUpdate | 0.00021         |
+                cryptoTransfer | 0.00010         |
+       cryptoGetAccountRecords | 0.00010         |
+          cryptoGetAccountInfo | 0.00010         |
+                  cryptoDelete | 0.01122         |
+```
+
+### Scenario 3 : Updating Throttles
+To update throttles put on services.
+
+Run the following command to download the `application.properties` file.
+```
+$ docker run -v $(pwd):/launch gcr.io/hedera-registry/yahcli:0.0.1 -p 2 -n previewnet sysfiles download props
+```
+
+Human Readable `application.properties` file will be downloaded to `previewnet/sysfiles/` folder.
+Update the required throttles in the properties file and perform a sysfile upload operation.
+
+```
+$ docker run -v $(pwd):/launch gcr.io/hedera-registry/yahcli:0.0.1 -p 2 -n previewnet sysfiles upload props
+```
+
+### Scenario 4 : Enable/Disable a Service
+To Enable/Disable permission for an operation.
+
+Run the following command to download the `api-permission.properties` file.
+```
+$ docker run -v $(pwd):/launch gcr.io/hedera-registry/yahcli:0.0.1 -p 2 -n previewnet sysfiles download permissions
+```
+
+Human Readable `api-permission.properties` file will be downloaded to `previewnet/sysfiles/` folder.
+Update the file by Enabling or Disabling a service and perform a sysfile upload operation. 
+
+```
+$ docker run -v $(pwd):/launch gcr.io/hedera-registry/yahcli:0.0.1 -p 2 -n previewnet sysfiles upload permissions
+```
+
+### Scenario 5 : Update AddressBook
+To Update address book of a network.
+
+```
+$ docker run -v $(pwd):/launch gcr.io/hedera-registry/yahcli:0.0.1 -n previewnet -p 2 sysfiles download help
+
+Download a system file
+      <sysfiles>...   system file names ('book', 'details', 'rates', 'fees',
+                        'props', 'permissions')
+                       or numbers
+                      or 'all' to download all system files
+  -d, --dest-dir=destination directory
+
+  -t, --type=If downloading protobuf v0.13.0 type file, mention if you want to 
+        download the `full` version [AddressBook] or `small` version 
+        [AddressBookForClients]
+                      One of
+                       Protobuf message names ['AddressBook',
```

Comment:
This can be `Protobuf message names options include` ?

Reply:
Updated this to `Protobuf Messages`